src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/jpa/repository/BasicUserDetailsRepository.kt

```diff
@@ -0,0 +1,49 @@
+package uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa.repository
+
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa.AccountStatus
+import java.util.Optional
+
+@Repository
+interface BasicUserDetailsRepository : JpaRepository<BasicUserPersonalDetail, String> {
+
+  @Query(
+    value =
+    " SELECT sua.USERNAME as username, sua.WORKING_CASELOAD_ID as activeCaseloadId, du.ACCOUNT_STATUS as accountStatus, sua.STAFF_ID as staffId, sm.FIRST_NAME as firstName, sm.LAST_NAME as lastName" +
+      " FROM STAFF_USER_ACCOUNTS sua " +
+      "      LEFT JOIN  DBA_USERS du  ON sua.USERNAME = du.USERNAME  " +
+      "     JOIN STAFF_MEMBERS sm ON sm.STAFF_ID = sua.STAFF_ID  " +
+      "WHERE " +
+      "     sua.USERNAME= ?1",
+    nativeQuery = true,
+  )
+  fun findBasicUserDetails(username: String): Optional<BasicUserPersonalDetail>
```

Comment:
Could we just call this `find` (given that the class name should be descriptive enough of what we're finding)?